manifests/f/FiloSottile/mkcert/1.4.4/FiloSottile.mkcert.version.yml

```
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://github.com/microsoft/winget-cli/raw/master/schemas/JSON/manifests/v1.2.0/manifest.version.1.2.0.json
+PackageIdentifier: FiloSottile.mkcert
+PackageVersion: 1.4.4
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: "1.2.0"
```

```suggestion
ManifestVersion: 1.2.0

```
Newline preferred at end of file